src/helpers.js

```diff
@@ -319,6 +320,39 @@ export async function processESLintMessages(messages, textEditor, showRule, work
   }))
 }
 
+/**
+ * Update the list of fixable rules
+ */
+function updateFixableRules() {
```

Comment:
What do you think about this function taking `fixableRules` as an argument, to minimize the side effects and avoid mutating something outside of its own scope?  I realize that if you pass it in and modify it, that's still technically a side effect, but it feels slightly cleaner.